CHANGELOG.develop

```diff
@@ -836,6 +919,7 @@ is enabled
 - Improve =magic-mode-alist= regular expression to detect =react= files
   (thanks to Jan Zdráhal)
 - Use JSX header without breaking React (thanks to Jam Risser)
+- Make auto-completion triggers more friendly (thanks to Cheng,Rong)
```

Comment:
Cheng Rong?

Not see other people seperating their family name.

Reply:
That's what is written in the commit message. Could be either way, so I'd rather leave it as is.